packages/datadog_flutter_plugin/e2e_test_app/integration_test/test_utils.dart

```diff
@@ -42,14 +42,19 @@ Future<void> initializeDatadog([DatadogConfigCallback? configCallback]) async {
   await DatadogSdk.instance.initialize(configuration);
 }
 
-Future<void> measure(String resourceName, AsyncVoidCallback callback) async {
-  var span = DatadogSdk.instance.traces?.startRootSpan(
-    'perf_measure',
-    resourceName: resourceName,
-    tags: {'operating_system': Platform.operatingSystem},
-  );
+Future<void> measure(String resourceName, AsyncVoidCallback callback,
+    [double targetSeconds = 0.02]) async {
+  var stopwatch = Stopwatch();
+  stopwatch.start();
   await callback();
-  span?.finish();
+  stopwatch.stop();
+  // ignore: unused_local_variable
+  final elapsedSeconds = stopwatch.elapsedMicroseconds / 1000000.0;
+  // TODO: Determine best way to monitor this moving forward
+  if (elapsedSeconds > targetSeconds) {
```

Comment:
How precise is `Stopwatch`? Say we have 21 ms vs expected 20 ms, can we see that often?

Reply:
My guess is that it's at least accurate to the microsecond